frontend/src/SearchChecklists.tsx

```diff
@@ -0,0 +1,59 @@
+import { RouteComponentProps } from '@reach/router';
+import React, { useContext } from 'react';
+import { navigate } from '@reach/router';
+import { Typography, Grid, Button, Paper, List, ListItem } from '@material-ui/core';
+import { Meta } from './types';
+import { AppContext } from './context';
+import { getCategoryName } from './utils/moduleHelpers';
+import { useStyles } from './styles';
+
+interface SearchChecklistsProps extends RouteComponentProps {
+}
+
+export const SearchChecklists = (props: SearchChecklistsProps) => {
+  const { categories } = useContext(AppContext);
+  const classes = useStyles({});
+  return (
+    <React.Fragment>
+      <Grid container spacing={5}>
+        <Grid item xs={12}>
+          <Typography variant="h4" gutterBottom>
+          Listo Checklists
+          </Typography>
+        </Grid>
+      </Grid>
+      <Grid container spacing={5}>
+        <Grid item xs={12}>
+          <Typography variant="body1" gutterBottom>
+            Search for and view individual checklists within Listo.
+          </Typography>
+        </Grid>
+        {Object.entries(categories).map(([categoryName, categoryModules]) => {
```

Comment:
This is a fairly complex bit of React. I would consider splitting out the JSX that the mapping function spits out into its own component